Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/azavea/postgis:latest
+MAINTAINER Azavea
+
+RUN apt-get update && apt-get install -y unzip postgis \
```

Comment:
This is a good thought. However after talking to @hectcastro I've consolidated all of the build steps in one `Dockerfile` `RUN` line.